Packs/GoogleCloudSCC/Integrations/GoogleCloudSCC/GoogleCloudSCC.py

```diff
@@ -0,0 +1,1125 @@
+            create_time = finding.get("createTime")
+
+            json_data["custom"] = prepare_markdown_fields_for_fetch_incidents(json_data)
+            json_data["finding_url"] = GoogleNameParser.get_finding_url(finding_name)
+
+            incidents.append({
+                "name": "{} - {}".format(INCIDENT_NAME_PREFIX, incident_name),
+                "occurred": create_time,
+                "rawJSON": json.dumps(json_data),
+                "details": json.dumps(json_data)
+            })
+        except Exception as e:
+            demisto.error(ERROR_MESSAGES["INVALID_INCIDENT"].format(str(e)))
+
+    acknowledges = [ack_id for ack_id in acknowledges if ack_id]
+    return incidents, acknowledges
+
+
+""" COMMANDS """
+
+
+def test_module(params: Dict[str, Any]) -> None:
+    """
+    Test authentication using service json
+    """
+    # Basic validation on configuration parameter
+    validate_configuration_param(params)
+
+    # Validate Service Account JSON and Organization ID
+    validate_service_account_and_organization_name(params)
+    if params.get('isFetch', False):
+        # Validate Project ID and Subscription ID.
+        validate_project_and_subscription_id(params)
+    demisto.results("ok")
+
+
+def fetch_incidents(client: GooglePubSubClient, params: Dict[str, Any]) -> Optional[list]:
+    """
+    Prepares incidents from past activity in Google Drive.
+
+    :param client: GooglePubSubClient object
+    :param params: arguments for fetch-incident.
+
+    :return: incidents (``List[dict]``): List of incidents that will be created in XSOAR.
+    """
+    validate_configuration_param(params)
+    max_messages = params.get("max_fetch") or DEFAULT_MAX_FETCH_VALUE
+
+    messages = client.pull_messages(max_messages=max_messages)
+    incidents, acks_id = convert_messages_to_incidents(messages)
+    if acks_id:
+        client.acknowledge_messages(acks_id)
+    return incidents
+
+
+@logger
+def asset_list_command(client: GoogleSccClient, args: Dict) -> CommandResults:
+    """
+    Lists an organization's assets.
+    :param client: SccClient Object.
+    :param args: Command argument(s).
+    :return: CommandResults object with context and human-readable.
+    """
+    # To validate arguments.
+    page_size = validate_get_int(args.get("pageSize"), ERROR_MESSAGES["INVALID_PAGE_SIZE_ERROR"],
+                                 MAX_PAGE_SIZE) or DEFAULT_PAGE_SIZE
+
+    resource_type = args.get("resourceType", "")
+    project = args.get("project", "")
+    field_mask = args.get("fieldMask", None)
+    order_by = args.get("orderBy", None)
+    active_assets_only = args.get("activeAssetsOnly", "false")
+    filter_string = args.get("filter", "")
+    read_time = args.get("readTime", None)
+    compare_duration = args.get("compareDuration", None)
+    page_token = args.get("pageToken", None)
+
+    # Creating filter
+    filter_string = create_filter_list_assets(resource_type, project, filter_string, active_assets_only)
```

Comment:
add a debug log after generating each filter_string
```suggestion
    filter_string = create_filter_list_assets(resource_type, project, filter_string, active_assets_only)
    demisto.debug(f'running command using the following filter: {filter_string}')
```

Reply:
Done